backend/akr/src/main/java/fi/oph/akr/scheduled/StatisticsGeneratorScheduled.java

```diff
@@ -0,0 +1,31 @@
+package fi.oph.akr.scheduled;
+
+import fi.oph.akr.service.StatisticsService;
+import fi.oph.akr.util.SchedulingUtil;
+import javax.annotation.Resource;
+import net.javacrumbs.shedlock.spring.annotation.SchedulerLock;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+@Component
+public class StatisticsGeneratorScheduled {
+
+  private static final Logger LOG = LoggerFactory.getLogger(StatisticsGeneratorScheduled.class);
+  private static final String LOCK_AT_LEAST = "PT0S";
+
+  private static final String LOCK_AT_MOST = "PT1H";
+
+  @Resource
+  private StatisticsService statisticsService;
+
+  @Scheduled(cron = "0 0 4 30 * *")
```

Comment:
Toimiiko tää cron-sääntö? https://crontab.guru näyttää että olisi epävalidi -> 30 osuisi `month`-positioon.
Ilmeisesti ideana kuitenkin ajaa päivittäin klo 04:30?

Reply:
Aivan, väärinhän se meni vaikka tätä on ees taas pyöritellyt devatessa. Laitan ajastuksen päivittäin 8:29 koska ilmeisesti osa testiympäristöistä tullaan sammuttamaan yöksi, eikä tämän ajoajalla oikeasti ole merkitystä.